api/src/helpers.py

```diff
@@ -37,6 +38,32 @@ def trigger_discover(input: Dict, data_pipeline_arn: str) -> Dict:
     )
 
 
+def execute_dag(env_name: str, input: Dict, dag_id: str) -> requests.Response:
+    assert dag_id in ("veda_discover", "veda_ingest")
+
+    client = boto3.client("mwaa")
+    token = client.create_cli_token(Name=env_name)
```

Comment:
I'm guessing that somewhere within the CDK code, it will be necessary to ensure that the service has permissions to make this call to the MWAA service.

Reply:
Indeed! @alukach also i would benefit tremendously if you can pair with me to show me how i could test this (or integration tests?) locally and/or in the dev environment.

Reply:
Hey, I just saw this now @vlulla  Let's definitely sync if it's still of value for you. I'll ping you over Slack.